homeassistant/components/deako/const.py

```diff
@@ -0,0 +1,14 @@
+"""Constants for Deako."""
+# Base component constants
+NAME = "Deako"
+DOMAIN = "deako"
+DOMAIN_DATA = f"{DOMAIN}_data"
+
+# Icons
+ICON = "mdi:format-quote-close"
+
+# Platforms
+LIGHT = "light"
+PLATFORMS = [LIGHT]
```

Comment:
Can we move this to `__init__.py` and use `Platform.LIGHT`?

Reply:
Yeah I don't even think this is used. Will remove.